src/sentry/integrations/msteams/notifications.py

```diff
@@ -39,10 +39,34 @@
     ResolvedActivityNotification,
     ResolvedInReleaseActivityNotification,
     ReleaseActivityNotification,
+    # TODO: Implement the following types of notifications.
```

Comment:
Could you file a ticket for this instead?

Reply:
@AniketDas-Tekky sure, I'll do that.